connectors/sources/tests/test_mysql.py

```diff
@@ -245,8 +245,6 @@ async def test_fetch_documents():
     source.connection_pool.acquire.cursor = Cursor
     source.connection_pool.acquire.cursor.is_connection_lost = False
 
-    query = "select * from table"
```

Comment:
We are using this `query` variable  at line 252 `response = source._connect(query)` 
we can not remove this variable.

Please see the [reference](https://github.com/elastic/connectors-python/blob/e4500202993faf5c0a7f608d74d20ceb8b5cba28/connectors/sources/tests/test_mysql.py#L252)

Reply:
I see, I've removed it by mistake